gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenDataFrameAggregateSuite.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.sql
 
+import io.glutenproject.execution.GlutenHashAggregateExecTransformer
```

Comment:
there is no GlutenHashAggregateExecTransformer in CH backend, use  HashAggregateExecBaseTransformer

Reply:
Thanks for your comment! Just fixed in the new commit.